dm/pkg/utils/db.go

```diff
@@ -391,6 +391,40 @@ func GetServerUnixTS(ctx context.Context, db *sql.DB) (int64, error) {
 	return ts, err
 }
 
+// GetCharsetAndDefaultCollation gets charset and default collation map.
+func GetCharsetAndDefaultCollation(ctx context.Context, db *sql.DB) (map[string]string, error) {
```

Comment:
please use wrapped connection, not sql.DB

Reply:
ok